com.microsoft.java.debug.plugin/src/main/java/com/microsoft/java/debug/plugin/internal/ResolveClasspathsHandler.java

```diff
@@ -189,7 +191,10 @@ public void acceptSearchMatch(SearchMatch match) {
                     }
                     String location = entries[j].getLocation();
                     if (location != null) {
-                        resolved.add(location);
+                        // remove duplicate classpath
+                        if (!resolved.contains(location)) {
```

Comment:
The contains is not necessary.  https://docs.oracle.com/javase/7/docs/api/java/util/HashSet.html#add(E)

Reply:
Should we normalize the path?